_dashboards/admin-ui-index/sm-dashboards.md

```diff
@@ -30,6 +45,10 @@ Before you create an SM policy, you need to set up a repository for snapshots.
 ```
 1. Select the **Add** button.
 
+#### Delete a repository
+
+To delete a snapshot repository configuration, select the repository in the **Repositories** list, then select the **Delete** button.
```

Comment:
Change "in" to "from". Replace the second comma with "and". Change the second instance of "select" to "choose".